src/components/Card/Cards.tsx

```diff
@@ -0,0 +1,68 @@
+// import Button from "@material-ui/core/Button";
+import React from 'react';
+import Card from '@material-ui/core/Card';
+// import CardActions from "@material-ui/core/CardActions";
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  cardGrid: {
+    paddingTop: theme.spacing(8),
+    paddingBottom: theme.spacing(8),
+  },
+  card: {
+    height: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  cardMedia: {
+    paddingTop: '56.25%', // 16:9
+  },
+  cardContent: {
+    flexGrow: 1,
+  },
+}));
+interface CardProps {
```

Comment:
That's cool. The only thing, please move above useStyles. It's not smth important, just a pattern.